test/cpp/jitter_buffer_test.cpp

```
@@ -0,0 +1,339 @@
+    }
+    const size_t grownTarget = jb.targetDepth();
+    assert(grownTarget > audio_config::kJitterInitialDepth);
+
+    // Drain whatever stale frames remain from the priming phase so the
+    // re-fill below doesn't run into the new max-depth cap.
+    while (jb.popAny().has_value()) {
+    }
+
+    // Now drive enough underrun-free intervals to trigger a shrink. Keep the
+    // buffer always above target so pop() never underruns.
+    const size_t shrinkAfter = audio_config::kJitterShrinkAfterStableTicks /
```

Clang (14.0.6)

[warning] variable 'shrinkAfter' is not initialized

(cppcoreguidelines-init-variables)